packages/contracts-bedrock/snapshots/storageLayout/OptimismMintableERC20Factory.json

```diff
@@ -1 +1,30 @@
-[]
\ No newline at end of file
+[
+  {
+    "bytes": "1",
+    "label": "_initialized",
+    "offset": 0,
+    "slot": "0",
+    "type": "uint8"
+  },
+  {
+    "bytes": "1",
+    "label": "_initializing",
+    "offset": 1,
+    "slot": "0",
+    "type": "bool"
+  },
+  {
+    "bytes": "20",
+    "label": "bridge",
```

Comment:
I don't love that `bridge` shares a slot with the initialize variables but its not that big of a deal

Reply:
do you think maybe we should add a spacer before the definition of `bridge` in `OptimismMintableERC20Factory.json`? so that `bridge` is stored in the next slot. we could do a gap, but there's already a gap after the definition of `bridge`.